docs/audits/ad-request-from-page-start.md

```diff
@@ -24,6 +24,7 @@ one of:
 
 | Library              | Host                             | Path          |
 |----------------------|----------------------------------|---------------|
+| AMP Doubleclick      | `securepubads.g.doubleclick.net` | `/gampad/ads` |
```

Comment:
Doubleclick -> DoubleClick

All the DoubleClick products have been rebranded, though. Not sure if the AMP-specific branding was updated or not, but should this be Google Ad Manager or something instead?